System.IO.Abstractions.TestingHelpers/MockFileStream.cs

```diff
@@ -16,10 +17,15 @@ public enum StreamType
             TRUNCATE
         }
 
-        public MockFileStream(IMockFileDataAccessor mockFileDataAccessor, string path, StreamType streamType)
+        public MockFileStream(
+            IMockFileDataAccessor mockFileDataAccessor,
+            string path,
+            StreamType streamType,
+            FileOptions options)
```

Comment:
I'd prefer to have a new,  additional overload instead of changing this one. Despite not breaking users, that would also remove a lot of clutter from this PR :)

Reply:
Good call, overload added.